src/D-FSL-1.0.xml

```diff
@@ -0,0 +1,380 @@
+             insbesondere das Bereithalten zum Download in Datennetzen.</p>
+            <p>Rechtsinhaber: Der bzw. die Urheber oder sonstigen Inhaber der ausschließlichen Nutzungsrechte an dem
+             Programm.</p>
+            <p>Source Code: Die für Menschen lesbare, in Programmiersprache dargestellte Form des Programms.</p>
+            <p>Verändern: Jede Erweiterung, Kürzung und Bearbeitung des Programms, insbesondere Weiterentwicklungen.</p>
+            <p>Verbreiten: Die öffentliche Weitergabe körperlicher Vervielfältigungsstücke, insbesondere auf
+             Datenträgern oder in Verbindung mit Hardware.</p>
+            <p>Vollständiger Source Code: Der Source Code in der für die Erstellung bzw. die Bearbeitung benutzten Form
+             zusammen mit den zur Übersetzung und Installation erforderlichen Konfigurationsdateien und
+             Software-Werkzeugen, sofern diese in der benötigten Form nicht allgemein gebräuchlich (z.B.
+             Standard-Kompiler) oder für jedermann lizenzgebührenfrei im Internet abrufbar sind.</p>
+        </li>
+        <li>
+            <p>§ 1 Rechte</p>
+            <list>
+                <li>
+                <b>(1)</b>
+                <p>Sie dürfen das Programm in unveränderter Form vervielfältigen, verbreiten und öffentlich
+                 zugänglich machen.</p>
+                </li>
+                <li>
+                    <b>(2)</b>
+                    <p>Sie dürfen das Programm verändern und entsprechend veränderte Versionen vervielfältigen,
+                     verbreiten und öffentlich zugänglich machen. Gestattet ist auch die Kombination des Programms
+                     oder Teilen hiervon mit anderen Programmen.</p>
+                </li>
+                <li>
+                    <b>(3)</b>
+                    <p>Sie erhalten die Rechte unentgeltlich.</p>
+                </li>
+            </list>
+        </li>
+        <li>
+            <p>§ 2 Pflichten beim Vertrieb</p>
+            <list>
+                <li>
+                    <b>(1)</b>
+                    <p>Wenn Sie das Programm verbreiten oder öffentlich zugänglich machen, sei es in unveränderter oder
+                     veränderter Form, sei es in einer Kombination mit anderen Programmen oder in Verbindung mit
+                     Hardware, dann müssen sie mitliefern:</p>
+                </li>
+                <li>
+                    <b>1.</b>
+                    <p>alle Vermerke im Source Code und/oder Object Code, die auf diese Lizenz hinweisen;</p>
+                </li>
+                <li>
+                    <b>2.</b>
+                    <p>alle Vermerke im Source Code und/oder Object Code, die über die Urheber des Programms Auskunft geben;</p>
+                </li>
+                <li>
+                    <b>3.</b>
+                    <p>einen für den Empfänger deutlich wahrnehmbaren Hinweis auf diese Lizenz und die Internetadresse
+                     http://www.d-fsl.de;</p>
```

Comment:
Upstream is inconsistent here, using [no wrapping chars in the HTML][1] and [`<>` in the text][2].   I recommend using `<alt>` or `<optional>` to cover both cases, and have no opinion on which form should be canonical.

[1]: https://www.hbz-nrw.de/produkte/open-access/lizenzen/dfsl/deutsche-freie-software-lizenz
[2]: https://www.hbz-nrw.de/produkte/open-access/lizenzen/dfsl/D-FSL-1_0_de.txt/view